handlers.py

```diff
@@ -198,7 +198,11 @@ def get_all_responses_with_game(self, message, game):
         return ["%s would also play %s (%s)" % (message.author.display_name, game, len(game.get_available_players()))] + get_any_ready_messages(game)
 
     def get_all_responses_with_games(self, message, games):
-        last_would_plays_for_listed_games = set(db.get_last_would_play(game).game for game in games)
```

Comment:
Why not `set(wp.game for wp in (db.get_last_would_play(game) for game in games) if wp)`